### ISSUE TITLE
UI Like StackEdit

### ISSUE THREAD

**Issue by RoyiAvital** [cben/mathdown#171]
Hello,

In my opinion [StackEdit](https://github.com/benweet/stackedit) is the best Online MarkDown editor out there.

Any chance you use it as a starting point and improve from there?

By the way, your [Wiki Page `Math in MarkDown`](https://github.com/cben/mathdown/wiki/math-in-markdown) is a great resource.

**Comment by cben** [comment]
Hi, sorry for glacial response.
I don't have enough time to develop Mathdown on its own; instead I'd like to "upstream"/contribute some of its features into other projects — see #172 for rough intentions.

StackEdit is great.  Last I checked (around v4) its editor became a custom marvel of engineering based on ContentEditable, which frankly scares me :-)
All the experience (and code) I got developing Mathdown is around CodeMirror.  If I will manage to contribute to another editor, it'll probably be HackMD.io.
